source/_includes/common-tasks/backups_create.md

```diff
@@ -0,0 +1,8 @@
+### Making a Backup from the UI
```

Comment:
This is not as common as it looks; creating backups is not the same for every installation method. The described steps below apply to installation types that ship with a Home Assistant Supervisor.